scripts/lang-stat

```diff
@@ -53,7 +53,7 @@ parse() {
     cd tmp
     json=out.json
     ast=out.ast
-    "$tree_sitter_parser" "$input" | ydump > "$json"
+    "$tree_sitter_parser" "$input" > "$json"
     "$ocaml_parser" "$input" "$json" > /dev/null
   ) > tmp/parse.log 2>&1
 }
```

Comment:
Performance of the stat collection is still IO-bound. A further cheap optimization here would be to write to ramdisk. Hopefully we'll be able to remove the conversion to json completely.